src/utils.js

```diff
@@ -546,6 +546,20 @@ function makePlural(num, string) {
   return `${num} ${string}`;
 }
 
+/**
+ * Pluralizes a word that typically ends with s based on the benchmark passed
+ *
+ * @param textToPlural
+ * @param pluralBenchmark
+ * @param punctuation
+ * @returns {string}
+ */
+const pluralText = (
```

Comment:
[inform] I think what you have here is fine, but just as a heads up, it looks like there is some support for defining plural messages within the message format (ICU syntax, see https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format) or perhaps `FormattedPlural` ([docs](https://formatjs.io/docs/react-intl/components/#formattedplural)).

Reply:
Leaving comment open as reference 👍🏽